akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +98,327 @@ Scala
+described there.
+
+### Auto-downing (DO NOT USE)
+
+There is an automatic downing feature that you should not use in production. For testing you can enable it with configuration:
+
+```
+akka.cluster.auto-down-unreachable-after = 120s
+```
+
+This means that the cluster leader member will change the `unreachable` node
+status to `down` automatically after the configured time of unreachability.
+
+This is a naïve approach to remove unreachable nodes from the cluster membership.
+It can be useful during development but in a production environment it will eventually breakdown the cluster. 
+When a network partition occurs, both sides of the partition will see the other side as unreachable and remove it from the cluster.
+This results in the formation of two separate, disconnected, clusters (known as *Split Brain*).
+
+This behaviour is not limited to network partitions. It can also occur if a node
+in the cluster is overloaded, or experiences a long GC pause.
+
+@@@ warning
+
+We recommend against using the auto-down feature of Akka Cluster in production. It
+has multiple undesirable consequences for production systems.
+
+If you are using @ref:[Cluster Singleton](cluster-singleton.md) or @ref:[Cluster Sharding](cluster-sharding.md) it can break the contract provided by 
+those features. Both provide a guarantee that an actor will be unique in a cluster.
+With the auto-down feature enabled, it is possible for multiple independent clusters
+to form (*Split Brain*). When this happens the guaranteed uniqueness will no
+longer be true resulting in undesirable behaviour in the system.
+
+This is even more severe when @ref:[Akka Persistence](persistence.md) is used in
+conjunction with Cluster Sharding. In this case, the lack of unique actors can 
+cause multiple actors to write to the same journal. Akka Persistence operates on a
+single writer principle. Having multiple writers will corrupt the journal
+and make it unusable. 
+
+Finally, even if you don't use features such as Persistence, Sharding, or Singletons, 
+auto-downing can lead the system to form multiple small clusters. These small
+clusters will be independent from each other. They will be unable to communicate
+and as a result you may experience performance degradation. Once this condition
+occurs, it will require manual intervention in order to reform the cluster.
+
+Because of these issues, auto-downing should **never** be used in a production environment.
+
+@@@
+
+### Leaving
+
+There are two ways to remove a member from the cluster.
+
+1. You can stop the actor system (or the JVM process, for example a SIGTERM sent from the environment). It will be detected
+as unreachable and removed after the automatic or manual downing.
+1. A more graceful exit can be performed if you tell the cluster that a node shall leave. 
+This can be performed using @ref:[JMX](../additional/operations.md#jmx) or @ref:[HTTP](../additional/operations.md#http).
```

Comment:
Put the graceful "normal" way up top and the crash one second?

Probably also a good idea to recommend gracefully leaving when possible since that can give faster handover etc?